### DISCUSSION TITLE
Get current boxPage code with twig ?

### DISCUSSION THREAD

**Discussion by chrisvidal** [OFFLINE-GmbH/oc-boxes-support#184] [Help] [answered]
Previously, we could get the page id inside a layout like this `page-{{this.page.id}}`

since all my pages are displayed automatically, how to get the page code in my layout please ?

**Comment by tobias-kuendig** [comment, marked as answer]
The `boxesPage` variable is available everywhere, so `{{ boxesPage.id }}` is probably what you want.

There are also other examples in the docs on how to use the variable: https://docs.boxes.offline.ch/concepts/box-pages.html#accessing-page-information-in-twig

**Reply by chrisvidal** [reply]
Thanks @tobias-kuendig  exactly what I needed `{{ boxesPage.slug }}` to get access to the developper code.
would be useful to put it in the documentation.
